app/scripts/metamask-controller.js

```diff
@@ -2849,6 +2864,9 @@ export default class MetamaskController extends EventEmitter {
     // The vault should not be exposed to the UI
     delete flatState.vault;
 
+    // The FCM should not be exposed to the UI
+    delete flatState.fcmToken;
```

Comment:
I think we can avoid having to do this. If the `PushPlatformNotificationsController` is added to `this.store` in the metamask-controller but not `this.memStore`, then it's state won't be exposed to the UI, but it will still be persisted.

Reply:
Ok, thanks for the comment, now everything is much clearer. Removed. Thx!